builtin/providers/aws/resource_aws_autoscaling_group_test.go

```diff
@@ -445,15 +445,6 @@ func TestAccAWSAutoScalingGroup_ALB_TargetGroups(t *testing.T) {
 						"aws_autoscaling_group.bar", "target_group_arns.#", "1"),
 				),
 			},
-
-			resource.TestStep{
-				Config: testAccAWSAutoScalingGroupConfig_ALB_TargetGroup_pre,
-				Check: resource.ComposeAggregateTestCheckFunc(
-					testAccCheckAWSAutoScalingGroupExists("aws_autoscaling_group.bar", &group),
-					resource.TestCheckResourceAttr(
-						"aws_autoscaling_group.bar", "target_group_arns.#", "0"),
-				),
-			},
```

Comment:
I'm not sure if it's a good idea just to remove a failing test here. 🤔 

Looking at the failure

```
=== RUN   TestAccAWSAutoScalingGroup_ALB_TargetGroups
--- FAIL: TestAccAWSAutoScalingGroup_ALB_TargetGroups (38.42s)
    testing.go:280: Step 3 error: Check failed: 1 error(s) occurred:
        
        * Check 2/2 error: aws_autoscaling_group.bar: Attribute 'target_group_arns.#' expected "0", got "1"
```
I suspect this may be caused by the fact that we don't randomize ALB target group names and these in turn remain attached to ASGs whilst other tests are running in parallel. i.e. how do you feel about randomizing the names there instead?